packages/material-ui/src/Tabs/Tabs.test.js

```diff
@@ -295,6 +295,23 @@ describe('<Tabs />', () => {
       expect(handleChange.args[0][1]).to.equal(1);
     });
 
+    it('should not call onChange when already selected', () => {
+      const handleChange = spy();
+      const { getAllByRole } = render(
+        <Tabs value={0} onChange={handleChange}>
+          <Tab />
+          <Tab />
+        </Tabs>
+      );
+
+      fireEvent.click(getAllByRole('tab')[1]);
+      expect(handleChange.callCount).to.equal(1);
+      expect(handleChange.args[0][1]).to.equal(1);
```

Comment:
This part is  already covered by the previous test ("should call onChange when clicking"). You can safely remove it.

Reply:
Done